skycultures/Book_of_Fixed_Stars_English/constellationship.fab

```diff
@@ -0,0 +1,102 @@
+﻿#UMi

```

Comment:
Again, abbreviations, and newline at end.

Reply:
done